db/error_handler_fs_test.cc

```diff
@@ -1874,6 +1877,8 @@ TEST_F(DBErrorHandlingFSTest, FLushWritNoWALRetryableErrorAutoRecover2) {
   Destroy(options);
 }
 
+// Auto resume fromt he flush retryable IO error. Activate the FS before the
```

Comment:
"fromt he" -> "from the". Likewise with the following two comments.

Reply:
Thanks for the review! Will change accordingly.